### PR TITLE
remove log noise

### PR DESCRIPTION
**Pull request by oberstet**

### CODE REVIEW

File: rlp/codec.py [hunk 1]
```diff
@@ -22,10 +22,6 @@
     from rlp.atomic import (
         Atomic,
     )
-    logger = logging.getLogger()
```
Comment (cburgdorf) [inline rlp/codec.py:25]: I think if you change this to `logging.getLogger("rlp.codec")` then it should be possible to silence this specific logger in downstream applications.

Comment (oberstet) [reply rlp/codec.py:25]: ok, changed. note: IMO the log level "warning" isn't best, but it should be "debug". eg when running under PyPy, I do not want to use the rust thing .. I suspect this is CPyExt based .. if so, it is likely slower than just letting the JIT do its job ..

Comment (cburgdorf) [reply rlp/codec.py:25]: > suspect this is CPyExt based

It isn't. It's pure rust and exposes Python bindings through [pyo3](https://github.com/PyO3/pyo3). It's quite a lot faster. You can run the [benchmarks](https://github.com/cburgdorf/rusty-rlp#benchmarks) locally to check it against your own machine.

**Review by cburgdorf — APPROVED** [review]

File: rlp/codec.py (hunk 1 above)
Comment (oberstet) [reply rlp/codec.py:25]: > Building with PyPy is also possible (via cpyext) for Python 3.6

https://github.com/PyO3/pyo3#usage

Comment (oberstet) [reply rlp/codec.py:25]: so a rust extension that uses cffim maybe. a cpyext extension, I won't use it unless being forced to. but in any case, thanks for reviewing! the PR fixes my itch;)

Comment (cburgdorf) [reply rlp/codec.py:25]: >so a rust extension that uses cffim maybe

Yep.

>I won't use it unless being forced to.

That's fair. We had to do this to optimize Trinity. It improves block processing time significantly. 

>the PR fixes my itch;)

Can you confirm that the fix works and you are able to silence the logger as expected? Would be nice to get this confirmed before I cut the release.

Comment (oberstet) [reply rlp/codec.py:25]: Yes, I can confirm, this PR silences the messages (with the default log level)

Comment (cburgdorf) [reply rlp/codec.py:25]: Thanks! I'll make sure to cut a patch release one of these days.

Comment (cburgdorf) [reply rlp/codec.py:25]: Here it is https://pypi.org/project/rlp/2.0.1/

**Pull request merged**